content/network/drivers/bridge.md

```diff
@@ -245,9 +241,20 @@ Restart Docker for the changes to take effect.
 
 ### Use IPv6 with the default bridge network
 
-If you configure Docker for IPv6 support (see [Use IPv6](#use-ipv6)), the
-default bridge network is also configured for IPv6 automatically. Unlike
-user-defined bridges, you can't selectively disable IPv6 on the default bridge.
+IPv6 can be enabled for the default bridge using options in `daemon.json`.
+
+```json
+{
+  "ipv6": true,
+  "bip": "192.168.1.1/24",
```

Comment:
I think we shouldn't add unrelated to parameters to this example (ie. `bip`, `fixed-cidr`, `mtu`, `default-gateway`, `dns`).

Reply:
It was an extension of the example above (which previously mixed v6 params with the other stuff) ... but it probably is clearer without - removed.